### DISCUSSION TITLE
Rework le switch de l'inventaire (v5.1 ?)

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#2572] [Nouvelles idées]
L'idée serait de pouvoir choisir l'item qu'on veut switch directement dans le /

Vu que les choix sont facultatif on pourrait en faire un (comme plusieurs ?)

Bon j'ai jamais codé la commande entièrement sur mon bot mais je peux le faire juste pour rajouter une image mieux détaillé

![Screenshot_20240829_001424](https://github.com/user-attachments/assets/9a5bf944-e742-40c9-aa8b-bccda227c218)

**Comment by symsym-1629** [comment]
ok je vois une option avec autocomplete en gros

**Reply by Bombafox** [reply]
Oui, att je te mets les screnn, j'ai copié mon inventaire pour l'idée.
![Screenshot_20240829_003451_edit_639084201204487](https://github.com/user-attachments/assets/4af41e7e-de31-4be0-9058-0e6e8e75602d)
![Screenshot_20240829_003523](https://github.com/user-attachments/assets/cef3baa7-1e67-497f-8762-138b0eaaf5f3)
![Screenshot_20240829_003551](https://github.com/user-attachments/assets/b3deea2d-dc11-48ae-a5b0-69b4f8e636a2)
![Screenshot_20240829_003603](https://github.com/user-attachments/assets/b3478fa8-28dd-4009-b537-d1b121d98545)

**Reply by Bombafox** [reply]
Faut partir du principe qu'on pourra voir 3 popo et 3 objets, et si ça ne casse pas le bot, pouvoir choisir de switch plusieurs item en même temps.
![Screenshot_20240829_004050](https://github.com/user-attachments/assets/203e4166-9d83-4c2a-ad44-f555e21f4c26)

**Comment by symsym-1629** [comment]
j'ai l'impression que ca fait trop d'options, surtout que tu switch jamais 2 trucs en mm temps honnêtement

**Comment by Cirpacha** [comment]
Du coup puisqu'il faut faire une suggestion pour que drapht lise (alors qu'il a créé un post sur le serv gngngngn), moi jveux un truc comme ça 
![image](https://github.com/user-attachments/assets/497cf046-69f9-4bfb-8b91-924f9056b734)

**Comment by BastLast** [comment]
j'ai du mal à comprendre la plu valu par rapport à un menu comme on a ?
